tcp_test.go

```diff
@@ -4,13 +4,26 @@ import (
 	"bytes"
 	"encoding/json"
 	"errors"
+	"os/user"
+	"strings"
 	"testing"
 
 	"github.com/AutoRoute/tuntap"
 
 	"github.com/AutoRoute/node/types"
 )
 
+func isRoot() bool {
```

Comment:
There's a function CheckRoot that error's if that user isn't root. Not exactly what we want here because we want to pass the test on Travis. This will also be a root test that needs to be tested locally even though it isn't an integration test.